README.md

```diff
@@ -50,12 +50,13 @@ If you want to simulate that your cache server is slow at incoming network
 upstream:
```

Comment:
Yeah, upstream latency is still possible, but it's not the default. We should definitely keep this example, but update the usage.